### PR TITLE
JBPM-4930 - Add warning to remove tab

### PR DESCRIPTION
**Pull request by cristianonicolai**
Included tests, I18 and removed using well inside popup body

### CODE REVIEW

File: uberfire-widgets/uberfire-widgets-commons/src/main/java/org/uberfire/ext/widgets/common/client/resources/i18n/CommonConstants.java [hunk 1]
```diff
@@ -132,4 +132,8 @@ String ConcurrentDelete( String identity,
     String Basic_Properties();
 
     String ClickToDisplay0(final String resourceGroup);
+
+    String RemoveTabTittle();
```
Comment (manstis) [inline uberfire-widgets/uberfire-widgets-commons/src/main/java/org/uberfire/ext/widgets/common/client/resources/i18n/CommonConstants.java]: Spelling mistake

**Comment by manstis** [comment]
Other than typo, +1 to merge.

**Comment by cristianonicolai** [comment]
@manstis fixed, thanks!

**Comment by manstis** [comment]
+1 to merge.

**Comment by csadilek** [comment]
0.8.x: f604c142719621a482b5297f302c3fda313f09c4

**Pull request closed without merging**